file/random_access_file_reader.cc

```diff
@@ -424,4 +424,12 @@ IOStatus RandomAccessFileReader::PrepareIOOptions(const ReadOptions& ro,
     return PrepareIOFromReadOptions(ro, SystemClock::Default().get(), opts);
   }
 }
+
+IOStatus RandomAccessFileReader::ReadAsync(
```

Comment:
@anand1976 Can you tell me what needs to handle in RandomAccessFileReader. In Read API, lot of stuff is happening so I wanted to confirm what is needed for this PR that needs to be addressed.

Reply:
For direct IO, we just need to call ```RandomAccessFileReader::Read``` and call the callback. For non-direct IO, just calling ```file_->ReadAsync``` should be fine for now. In a future PR, we should take care of stats, perf timers etc.